src/main/java/seedu/address/ui/PersonDetailedView.java

```diff
@@ -20,6 +23,7 @@ public class PersonDetailedView extends UiPart<Region> {
     private static final String FXML = "PersonDetailedView.fxml";
 
     public final Person person;
+    private final PersonDetailedViewContentManager contentManager;
```

Comment:
Since this is final why not just make it public

Reply:
I made it private as I wanted to make sure that it was used nowhere else except for here, could I check why you think public would be better? Similarly for the other comments regarding this.

Reply:
@Justin-Aw46 As learnt from CS2030S, part of OOP principles include information hiding. Public reveals the internal structure and possibly unexpected/unintended behaviour.